src/test/kotlin/no/nav/familie/ef/sak/iverksett/IverksettingDtoMapperTest.kt

```diff
@@ -107,11 +137,270 @@ internal class IverksettingDtoMapperTest {
+        assertThat(iverksettDto.søker.barn.first().termindato).isEqualTo(LocalDate.of(2022, 3, 25))
+        assertThat(iverksettDto.søker.tilhørendeEnhet).isEqualTo("4489")
+        assertThat(iverksettDto.søker.adressebeskyttelse?.name).isEqualTo(ADRESSEBESKYTTELSEGRADERING.UGRADERT.name)
+
+        val vedtak = iverksettDto.vedtak
+        assertThat(vedtak.beslutterId).isEqualTo("beslutter")
+        assertThat(vedtak.brevmottakere.size).isEqualTo(2)
+        assertThat(vedtak.brevmottakere[0].ident).isEqualTo("personIdent")
+        assertThat(vedtak.brevmottakere[0].navn).isEqualTo("fornavn etternavn")
+        assertThat(vedtak.brevmottakere[0].identType.name).isEqualTo(Brevmottaker.IdentType.PERSONIDENT.name)
+        assertThat(vedtak.brevmottakere[0].mottakerRolle.name).isEqualTo(MottakerRolle.BRUKER.name)
+
+        assertThat(vedtak.brevmottakere[1].ident).isEqualTo("organisasjonsnummer")
+        assertThat(vedtak.brevmottakere[1].navn).isEqualTo("organisasjonsnavn")
+        assertThat(vedtak.brevmottakere[1].identType.name).isEqualTo(Brevmottaker.IdentType.ORGANISASJONSNUMMER.name)
+        assertThat(vedtak.brevmottakere[1].mottakerRolle.name).isEqualTo(MottakerRolle.BRUKER.name)
+
+        assertThat(vedtak.opphørÅrsak).isNull() // Burde ha verdi i test?
+        assertThat(vedtak.resultat).isEqualTo(Vedtaksresultat.INNVILGET)
+        assertThat(vedtak.saksbehandlerId).isEqualTo("opprettetAv")
+
+        val tilbakekrevingMedVarsel = vedtak.tilbakekreving?.tilbakekrevingMedVarsel
+        assertThat(tilbakekrevingMedVarsel?.perioder?.size).isEqualTo(1)
+        assertThat(tilbakekrevingMedVarsel?.perioder?.first()?.fom).isEqualTo(LocalDate.of(2022, 3, 30))
+        assertThat(tilbakekrevingMedVarsel?.perioder?.first()?.tom).isEqualTo(LocalDate.of(2022, 3, 31))
+        assertThat(tilbakekrevingMedVarsel?.varseltekst).isEqualTo("varseltekst")
+        assertThat(tilbakekrevingMedVarsel?.sumFeilutbetaling).isEqualTo(BigDecimal("1000.0"))
+        assertThat(vedtak.tilbakekreving?.tilbakekrevingsvalg)
+                .isEqualTo(TilbakekrevingsvalgKontrakt.OPPRETT_TILBAKEKREVING_MED_VARSEL)
+
+
+        assertThat(vedtak.tilkjentYtelse?.startdato).isEqualTo(LocalDate.of(2022, 4, 7))
+        //opphørsdato ikke i bruk?
+
+        assertThat(vedtak.tilkjentYtelse?.andelerTilkjentYtelse?.size).isEqualTo(1)
+
+        assertThat(vedtak.vedtaksperioder.size).isEqualTo(1)
+        val vedtaksperiode = vedtak.vedtaksperioder.first()
+        assertThat(vedtaksperiode.fraOgMed).isEqualTo(LocalDate.of(2022, 3, 27))
+        assertThat(vedtaksperiode.tilOgMed).isEqualTo(LocalDate.of(2022, 3, 28))
+        assertThat(vedtaksperiode.aktivitet.name).isEqualTo(AktivitetType.BARN_UNDER_ETT_ÅR.name)
+        assertThat(vedtaksperiode.periodeType.name).isEqualTo(VedtaksperiodeType.HOVEDPERIODE.name)
+        //assertThat(iverksettDto.vedtak.vedtakstidspunkt) - sjekker ikke denne da det er LocalDate.now()
```

Comment:
Veldig fin test - og det er ikke mulig å teste _alle_ scenarier på en test, men kan godt ha en egen mappertest for 
- Opphør (men bare med opphør-spesifikke felter)
- Avslag (bare avslag-spesifikke felter)

Reply:
Er det egentlig vits når jeg uansett gjør en sjekk på alle felter i denne testen? Felt-verdiene stemmer uansett ikke overens logisk.
Når du nevnte det oppdaget jeg en ting. Det ene feltet jeg ikke har sjekket noen verdi på, som er opphørÅrsak, kan jeg ikke se får noen verdi noen steder i koden. Er det noe som burde blitt satt som følge av at vi nå har revurderinger? (Som nevnt i kommentar i koden, annotert med @Imporovement i IverksettingDtoMapper)

Reply:
Se der, ja. Så ikke at jeg hadde skrevet den kommentaren.